reach2

```diff
@@ -1,13 +1,19 @@
 #!/bin/sh
 
-case "$1" in
-  "bootstrap") echo "TODO" ;; # Document this in `optparse` as well
-
-  *)
-    s=$(docker run -it --rm reachsh/reach-cli:latest "$@")
-    if [ "$(echo "$s" | head -n1 | tr -d '\r')" = 'RUNME' ]
-      then echo "$s" | tr -d '\r' | sed 's/RUNME//' | /bin/sh
-      else echo "$s"
-    fi
-    ;;
+IMG='reachsh/reach-cli:latest'
+
+if ! (which make docker docker-compose >/dev/null 2>&1) ; then
+  echo "Reach relies on an installation of make, docker, and docker-compose"
+  exit 1
+fi
+
+if [ "$(docker image ls -q $IMG)" = '' ] ; then
```

Comment:
I bet shellcheck will want an `x` in there :)

Reply:
I've been running `shellcheck` all along and it has no objection to this line. Would you rather there be an `x` anyway?

Reply:
No, I'm good